sdk/src/signature.rs

```diff
@@ -61,16 +61,18 @@ impl Signature {
         Self(GenericArray::clone_from_slice(&signature_slice))
     }
 
+    pub(self) fn verify_verbose(
```

Comment:
You didn't introduce this but I would change the variable names here from using 'pubkey' to 'publickey' to make it clear that we are referring to an `ed25519_dalek::PublicKey` and not Solana's `Pubkey`